Shared/alerts/AlertConditionsController.cpp

```diff
@@ -1071,27 +1089,46 @@ AlertTarget* AlertConditionsController::targetFromItemIdAndIndex(int itemId, int
       if (!layer)
         continue;
 
-      FeatureLayer* featLayer = qobject_cast<FeatureLayer*>(layer);
-      if (!featLayer)
+      FeatureLayer *featLayer = qobject_cast<FeatureLayer*>(layer);
+      DynamicEntityLayer *dynamicEntityLayer = qobject_cast<DynamicEntityLayer*>(layer);
+      if (!featLayer && !dynamicEntityLayer)
         continue;
 
       currIndex++;
 
       if (currIndex == targetOverlayIndex)
       {
-        if (itemId == -1)
-        {
-          if (!m_layerTargets.contains(featLayer->name()))
-            m_layerTargets.insert(featLayer->name(), new FeatureLayerAlertTarget(featLayer));
-
-          targetDescription = featLayer->name();
-          return m_layerTargets.value(featLayer->name(), nullptr);
-        }
-        else
-        {
-          targetDescription = QString("%1 [%2]").arg(featLayer->name(), QString::number(itemId));
-          return targetFromFeatureLayer(featLayer, itemId);
-        }
+          if (featLayer)
+          {
+            if (itemId == -1)
+            {
+              if (!m_layerTargets.contains(featLayer->name()))
+                m_layerTargets.insert(featLayer->name(), new FeatureLayerAlertTarget(featLayer));
+
+              targetDescription = featLayer->name();
+              return m_layerTargets.value(featLayer->name(), nullptr);
+            }
+            else
+            {
+              targetDescription = QString("%1 [%2]").arg(featLayer->name(), QString::number(itemId));
+              return targetFromFeatureLayer(featLayer, itemId);
+            }
+          }
+          if (dynamicEntityLayer)
+          {
+              if (itemId == -1)
```

Comment:
Please use consistent indent size within the project.

Reply:
Disabled clang-formatting and using Qt team style file.